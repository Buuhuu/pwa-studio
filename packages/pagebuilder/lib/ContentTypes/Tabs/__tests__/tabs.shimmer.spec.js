import React from 'react';

import { createTestInstance } from '@magento/peregrine';

import TabsShimmer from '../tabs.shimmer';

jest.mock('@magento/venia-ui/lib/classify');

test('renders an empty TabsShimmer component', () => {
    const component = createTestInstance(<TabsShimmer />);

    expect(component.toJSON()).toMatchSnapshot();
});

test('renders a TabsShimmer component with minHeight', () => {
    const props = {
        minHeight: '400px'
    };
    const component = createTestInstance(<TabsShimmer {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
});
