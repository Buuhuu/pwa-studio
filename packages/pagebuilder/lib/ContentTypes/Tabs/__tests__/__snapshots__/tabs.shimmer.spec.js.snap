// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a TabsShimmer component with minHeight 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
>
  <ul
    className="navigation"
  >
    <li
      className="header"
    >
      <span>
         
      </span>
    </li>
  </ul>
  <div
    className="content"
    style={
      Object {
        "minHeight": "400px",
      }
    }
  />
</div>
`;

exports[`renders an empty TabsShimmer component 1`] = `
<div
  aria-busy="true"
  aria-live="polite"
  className="root_rectangle"
  style={
    Object {
      "marginBottom": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginTop": undefined,
      "paddingBottom": undefined,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
>
  <ul
    className="navigation"
  >
    <li
      className="header"
    >
      <span>
         
      </span>
    </li>
  </ul>
  <div
    className="content"
    style={
      Object {
        "minHeight": undefined,
      }
    }
  />
</div>
`;
